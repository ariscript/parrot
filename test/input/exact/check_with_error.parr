check:
    1 + true is ().

nil
