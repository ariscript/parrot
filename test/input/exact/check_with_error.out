error encountered while running test at 2:4-2:18: arithmetic expected number, recieved: true
nil
